e2e/src/orbs-client.ts

```diff
@@ -0,0 +1,81 @@
+import { types } from "orbs-common-library/src/types";
+import { CryptoUtils } from "orbs-common-library/src/CryptoUtils";
+import { grpc } from "orbs-common-library/src/grpc";
+import { delay } from "bluebird";
+
+
+type OrbsHardCodedContractMethodArgs = [string | number] | undefined[];
+
+export class OrbsHardCodedContractAdapter {
+    orbsSession: OrbsClientSession;
+    contractAddress: string;
+
+    constructor(orbsSession: OrbsClientSession, contractAddress: string) {
+        this.orbsSession = orbsSession;
+        this.contractAddress = contractAddress;
+    }
+
+    public async sendTransaction(methodName: string, args: OrbsHardCodedContractMethodArgs) {
+        const argumentsJson = JSON.stringify({
+            method: methodName,
+            args: args
+        });
+        return await this.orbsSession.sendTransaction(this.contractAddress, argumentsJson);
+    }
+
+    public async call(methodName: string, args: OrbsHardCodedContractMethodArgs) {
+        const argumentsJson = JSON.stringify({
+            method: methodName,
+            args: args
+        });
+        return this.orbsSession.call(this.contractAddress, argumentsJson);
+    }
+}
+
+export class OrbsClientSession {
+    keyPair: CryptoUtils;
+    orbsClient: types.PublicApiClient;
+    subscriptionKey: string;
+
+    constructor(keyPair: CryptoUtils, subscriptionKey: string, orbsClient: types.PublicApiClient) {
+        this.keyPair = keyPair;
+        this.orbsClient = orbsClient;
+        this.subscriptionKey = subscriptionKey;
+    }
+
+    async sendTransaction(contractAddress: string, argumentsJson: string) {
+        const signedTransaction = this.generateSignedTransaction(contractAddress, argumentsJson);
+
+        const res = await this.orbsClient.sendTransaction({
+            transaction: signedTransaction,
+            transactionAppendix: {
+                prefetchAddresses: [],
+                subscriptionKey: this.subscriptionKey
+            }
+        });
+        await delay(10000);
```

Comment:
It's encapsulated for reason. I decided to make this call blocking, and that the caller should  assume that the transaction has been processed, BUT it will be eventually replaced by a more appropriate solution that actually verifies it.